cli_reference/openshift_cli/managing-cli-profiles.adoc

```diff
@@ -0,0 +1,346 @@
+[id="managing-cli-profiles"]
+= Managing CLI profiles
```

Comment:
All of this content must be modularized.

Switching between CLI profiles, Manually configuring CLI profiles, Loading and merging rules should all be separate module files. For more guidance on this process, please see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/contributing_user_stories.adoc#how-do-i-write-in-modules

Note that when the content is restructured into the module format, xrefs within modules are currently forbidden due to known tooling issues: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

Reply:
> @Amrita42 Thanks for this update! Overall, this content needs to be updated with modularization and with latest 4.x style guidelines applied. Please let me know if you have any questions. Thanks!

Thanks @ahardin-rh for the useful feedback. I have re-worked the content and created conceptual modules as per mod doc/OCP guidelines. PTAL at the new preview.